main/main.cpp

```diff
@@ -2468,8 +2485,8 @@ Error Main::setup2() {
 	Thread::make_main_thread(); // Make whatever thread call this the main thread.
 	set_current_thread_safe_for_nodes(true);
 
-	// Print engine name and version
-	Engine::get_singleton()->print_header(String(VERSION_NAME) + " v" + get_full_version_string() + " - " + String(VERSION_WEBSITE));
+	// Don't use rich formatting to prevent ANSI escape codes from being written to log files.
+	print_header(false);
```

Comment:
This is a workaround for https://github.com/godotengine/godot/pull/36252#issuecomment-1924194109, which we could remove once we have support for stripping ANSI escape codes in RotatedFileLogger.